aten/src/ATen/Context.h

```diff
@@ -477,19 +479,6 @@ static inline void manual_seed(uint64_t seed) {
     }
   }
 
-  const auto xpu_num_gpus = detail::getXPUHooks().getNumGPUs();
```

Comment:
These codes will be added with PR related to `Generator` as they affect `torch.manual_seed` in CI.